docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md

```diff
@@ -114,7 +114,18 @@ information, refer to [Enable Local Harbor Registry](./local-registry.md).
    [Upload Cluster Images to External Registry with Palette Edge CLI](./upload-images-to-registry.md) to learn how to
    use the Palette Edge CLI to upload all images in a cluster profile to an external registry.
 
-9. Follow the [Create Cluster Definition](../site-installation/cluster-deployment.md) guide and deploy your cluster.
+9. In the Kubernetes layer of your cluster profile, remove `AlwaysPullImages` from
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins` and add it to
+   `cluster.config.clusterConfiguration.apiServer.extraArgs.enable-admission-plugins`.
+
+   The resulting layer configuration should look like the following.
+
+   ```yaml
+   disable-admission-plugins: AlwaysPullImages,AlwaysAdmit
```

Comment:
@brathina-spectro Do you have to explicitly add `AlwaysPullImages` in `disable-admission-plugins`?

From what I recollect, we just used to remove `AlwaysPullImages` from `enable-admission-plugins`?

Reply:
`AlwaysPullImages` admission plugin should be removed from enable-admission-plugin. Not changes are needed in disable-admission. You can refer what we've documented already for [airgap use case](https://docs.spectrocloud.com/integrations/harbor-edge/#prerequisites)